packages/teams-js/test/private/bot.spec.ts

```diff
@@ -178,74 +186,88 @@ describe('bot', () => {
+        const handleError = (_error: string): any => (error = _error);
+
+        // send message request
+        bot.authenticate(request, handleAuth, handleError);
+
+        // find message request in jest
+        const message = utils.findMessageByFunc('bot.authenticate');
+
+        // check message is sending correct data
+        expect(message).not.toBeUndefined();
+        expect(message.args).toContain(request);
+
+        // simulate response
+        const data = {
+          success: true,
+          response: { data: ['some', 'queried', 'items'] },
+        };
+
+        utils.respondToMessage(message, data.success, data.response);
+
+        // authenticate should also return data because first query
+        expect(botResponse).toEqual({ data: ['some', 'queried', 'items'] });
+        expect(error).toBeUndefined();
+      });
+
+      it(`bot.authenticate should invoke error callback on unauthorized with ${context} context`, async () => {
+        await utils.initializeWithContext('content');
+        const request = {
+          query: '',
+          commandId: 'someCommand',
+          url: 'someUrl',
+        };
+
+        let botResponse: bot.Results;
+        let error: string;
+
+        const handleBotResponse = (response: bot.Results) => (botResponse = response);
+        const handleError = (_error: string): any => (error = _error);
+
+        bot.authenticate(request, handleBotResponse, handleError);
+        const message = utils.findMessageByFunc('bot.authenticate');
+        expect(message).not.toBeUndefined();
+        expect(message.args).toContain(request);
+
+        // simulate response
+        const data = {
+          success: false,
+          response: 'Bot authorization was unsuccessful',
+        };
+
+        utils.respondToMessage(message, data.success, data.response);
+
+        // check data is returned properly
+        expect(error).toBe('Bot authorization was unsuccessful');
+        expect(botResponse).toBeUndefined();
+      });
+    });
   });
 
-  it('should invoke error callback on unauthorized', async () => {
-    await utils.initializeWithContext('content');
-    const request = {
-      query: '',
-      commandId: 'someCommand',
-      url: 'someUrl',
-    };
-
-    let botResponse: bot.Results;
-    let error: string;
-
-    const handleBotResponse = (response: bot.Results) => (botResponse = response);
-    const handleError = (_error: string): any => (error = _error);
-
-    bot.authenticate(request, handleBotResponse, handleError);
-    const message = utils.findMessageByFunc('bot.authenticate');
-    expect(message).not.toBeUndefined();
-    expect(message.args).toContain(request);
-
-    // simulate response
-    const data = {
-      success: false,
-      response: 'Bot authorization was unsuccessful',
-    };
-
-    utils.respondToMessage(message, data.success, data.response);
+  describe('Testing bot.isSupported function', () => {
```

Comment:
nit: Don't think we need function in here. Testing `bot.isSupported` should be enough

Reply:
We are trying to change the standard format so that we can group them together and be informative in JEST CLI